### PR TITLE
Contact section

### PR DESCRIPTION
**Pull request by adel-gu**
## Project Requirements 
- Add Contact form.
- Used built-in `HTML5` validations.
- All input fields have placeholder and required attributes.
- Collect Data using `formspree service`.
- followed UX rules.

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @adel-gu, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a project, that doesn't have a good pull request description, kindly add one. See a demo on how to add a PR description below:

pull_request_desc
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

File: README.md [hunk 1]
```diff
@@ -27,4 +31,4 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is [MIT](./MIT.md) licensed.
\ No newline at end of file
+This project is [MIT](./MIT.md) licensed.
```
Comment (Qoosim) [inline README.md:34]: - [ ] Kindly add more content to your readme file. It still looks very scanty. Try to add more instructive information that give the users how to play around your portfolio. Information like how to clone it and run it on their PC.

File: styles/style.css [hunk 2]
```diff
@@ -260,3 +260,94 @@ button {
...
+.input-item:nth-child(2) {
+  margin-block: 0.9em;
+}
+
+.input-item label {
+  color: #1a1a20;
+  position: absolute;
+  left: 1em;
+  top: 0.25em;
+}
```
Comment (Qoosim) [inline styles/style.css:355]: - [ ] It is required to use media queries for the desktop responsiveness of your portfolio. Media queries are a key part of responsive web design, as they allow you to create different layouts depending on the size of the viewport.
- [ ] Kindly check below point from the project requirements.
     -  Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: 768px or wider).
- You can follow this [link](https://www.w3schools.com/css/css_rwd_mediaqueries.asp) to learn how to use media queries for the responsiveness of your portfolio. You can check out this [link](https://css-tricks.com/snippets/css/media-queries-for-standard-devices/) too

**Review by Qoosim — CHANGES_REQUESTED** [review]
## **Status: Required Changes** :recycle: 

Hi @adel-gu and @letarulinda, 

Good job so far! :clap: 
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Top Highlights :1st_place_medal: 

-  No linter errors. :heavy_check_mark: 
-  Professional Github Flow. :heavy_check_mark: 
-  Meaningful commit messages. :heavy_check_mark: 
-  HTML && CSS Best Practices. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] With what I saw on your desktop version of the project, you are a bit far from how it is supposed to appear. Kindly go back to the materials you were provided before this section of the project or follow the links that I provided in my last inline comment to make your project desktop responsive.

### Your Design Desktop Version

your-desktop-design

*****************************************************

## Figma Design Desktop Version 1

figma-design-1


### Figma Design Desktop Version 2

figma-design-2

### Figma Design Desktop Version 3

figma-design-3

### Figma Design Desktop Version 4

figma-design-4

- [ ] Kindly work your design to stick as much as possible to your chosen template from the provided Figma design templates. Thank you. Happy learning, happy coding. :bowing_man:

File: styles/style.css [hunk 3]
```diff
@@ -260,3 +260,96 @@ button {
...
+  padding-top: 6.625em;
+  display: flex;
+  flex-direction: column;
+}
+
+.social-links {
+  background: var(--bg-blue);
+  margin-top: 4em;
+  padding-top: 2em;
+}
```
Comment (Bondok6) [inline styles/style.css]: - [ ] According to the requirements You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9). So kindly change the background color to `#1A2236` to look similar to the original design 💯👌

| Your Design  | Figma Design  |   
| --- | --- |
|   |  |

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @adel-gu and @Qoosim 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✅
- Great Job using FormSpree ✅
- Your project is professional ✅
- Your repo is professional ✅
- Professional README ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles/style.css (hunk 3 above)
Comment (adel-gu) [reply styles/style.css]: Thank you @Bondok6, I'll do so.

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @adel-gu 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**